pkg/codegen/testing/test/testdata/functions-pp/dotnet/functions.cs

```diff
@@ -6,7 +6,9 @@
 {
     var encoded = Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes("haha business"));
 
-    var joined = string.Join("-", new[]
+	var decoded = Encoding.UTF8.GetString(Convert.FromBase64String(encoded));
+
+	var joined = string.Join("-", new[]
```

Comment:
The indentation is wrong here. 
```suggestion
    var decoded = Encoding.UTF8.GetString(Convert.FromBase64String(encoded));

    var joined = string.Join("-", new[]
```